test/testdata/container_redis.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
I'm not sure what we can do about it now, but I can imagine we could struggle keeping this in sync in the future...